cms/templates/product_page.html

```diff
@@ -93,5 +93,26 @@ <h2>What you’ll learn</h2>
           </div>
         </div>
       </section>
+      {% if page.faculty_members %}
+      <section class="faculty-section">
+        <div class="container">
+          <h2>{{ page.faculty_section_title }}</h2>
+          <ul class="faculty-members">
+            {% for member in page.faculty_members %}
+              <li>
+                <div class="member-card">
+                  <img src="{% wagtail_img_src member.value.image %}" alt="{{ member.name }}">
+                  <div class="member-info">
+                    <h2 class="name">{{ member.value.name }}</h2>
+                    <div class="designation">{{ member.value.designation }}</div>
```

Comment:
Unless I'm missing something, this field does not exist and users cannot enter anything for it